pyphare/pyphare/pharein/diagnostics.py

```diff
@@ -52,8 +52,9 @@ def validate_timestamps(clazz, **kwargs):
                 raise RuntimeError(f"Error: timestamp({sim.time_step_nbr}) cannot be greater than simulation.final_time({sim.final_time}))")
         if not np.all(np.diff(timestamps) >= 0):
             raise RuntimeError(f"Error: {clazz}.{key} not in ascending order)")
-        if not np.all(np.abs(timestamps / sim.time_step - np.rint(timestamps/sim.time_step) < 1e-10)):
-            raise RuntimeError(f"Error: {clazz}.{key} is inconsistent with simulation.time_step)")
+        if not np.all(np.abs(timestamps / sim.time_step - np.rint(timestamps/sim.time_step) < 1e-9)):
+            print(np.abs(timestamps / sim.time_step - np.rint(timestamps/sim.time_step)).max())
```

Comment:
needed? or maybe incorporated into the error string?

Reply:
it's a leftover from debugging... the 1e-10 was failing, had to put it to 1e-9... not sure why